libnd4j/CMakeLists.txt

```diff
@@ -22,8 +22,8 @@ if (SD_CUDA)
     enable_language(CUDA)
     if(CMAKE_CXX_COMPILER MATCHES "MSVC" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL "19.29" )
             message("Working around windows build failure with visual studio. Visual studio 16.10 introduced a compiler bug with compilng CUDA code with C++14. Set the cuda standard to 17 as a workaround.")
-            set(CMAKE_CXX_STANDARD 17)
-            set(CMAKE_CUDA_STANDARD 17)
+            set(CMAKE_CXX_STANDARD 11)
+            set(CMAKE_CUDA_STANDARD 11)
```

Comment:
Why does this even need to be changed? This should only set it for CUDA, where oneDNN isn't involved at all.

Reply:
@treo ah this was to test the compile and verifying different verisons of c++. I was still tracking down the onednn changes at the time. My problems were also only on windows. Great check. Reverted.